src/SDKs/Billing/Billing.Tests/Properties/AssemblyInfo.cs

```diff
@@ -16,5 +16,5 @@
 
 [assembly: ComVisible(false)]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
Please do not update the `AssemblyVersion`

Reply:
Reverted